java/ql/lib/ext/threatmodels/threat-model-grouping.model.yml

```diff
@@ -0,0 +1,25 @@
+extensions:
+
+  - addsTo:
+      pack: codeql/java-all
+      extensible: threatModelGrouping
+    data:
+      # Default threat model
+      - ["remote", "default"]
+      - ["uri-path", "default"]
+
+      # Android threat models
+      - ["android-widget", "android"]
+      - ["android-external-storage-dir", "android"]
+      - ["contentprovider", "android"]
+      - ["android-external-storage-dir", "android"]
```

Comment:
Is the duplication at lines 13 and 15 intentional?

Reply:
No - thank you 👍